packages/esm-patient-allergies-app/src/constants.ts

```diff
@@ -1 +1,3 @@
+export const ALLERGY_REACTION_CONCEPT = '162555AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
 export const patientAllergiesFormWorkspace = 'patient-allergy-form-workspace';
+export const allergiesToShowCount = 5;
```

Comment:
Matter of taste, but I'd just put this in the file it goes with. It's pretty specific to that component.